src/features/security-cloud-game/SecurityCloudGame.js

```diff
@@ -12,8 +13,8 @@ import { checkOverlap, getGridPosition, getRandomCloudPosition } from './utils';
 import './SecurityCloudGame.scss';
 
 // pick these numbers to balance accessibility and game complexity
-const CLOUD_SIZE_RATIO = 5;
-const GRID_TRACK_SIZE_RATIO = 20;
+const CLOUD_SIZE_RATIO = 4;
+const GRID_TRACK_SIZE_RATIO = 16;
```

Comment:
just curious, how were these values determined? are there any min / max size considerations?

Reply:
I queried our embed widget analytics data for the height and width being used and determined that these values work better across all sizes (330 x 150 - 2560 x 1600 +).